govc/tags/info.go

```diff
@@ -28,80 +28,80 @@ import (
 	"github.com/vmware/govmomi/vapi/tags"
 )
 
-type get struct {
+type info struct {
 	*flags.ClientFlag
 	*flags.OutputFlag
 	name string
 }
 
 func init() {
-	cli.Register("tags.get", &get{})
+	cli.Register("tags.info", &info{})
 }
 
-func (cmd *get) Register(ctx context.Context, f *flag.FlagSet) {
+func (cmd *info) Register(ctx context.Context, f *flag.FlagSet) {
 	cmd.ClientFlag, ctx = flags.NewClientFlag(ctx)
 	cmd.OutputFlag, ctx = flags.NewOutputFlag(ctx)
 	cmd.ClientFlag.Register(ctx, f)
 	cmd.OutputFlag.Register(ctx, f)
 	f.StringVar(&cmd.name, "n", "", "Name of category")
 }
 
-func (cmd *get) Process(ctx context.Context) error {
+func (cmd *info) Process(ctx context.Context) error {
 	if err := cmd.ClientFlag.Process(ctx); err != nil {
 		return err
 	}
 	return nil
 }
-func (cmd *get) Usage() string {
+func (cmd *info) Usage() string {
 	return "TAGID"
 }
 
-func (cmd *get) Description() string {
-	return `Get tags by tags' ID, or Get tags for category by tag name and category ID. 
+func (cmd *info) Description() string {
+	return `Get tags info by tags' ID, or Get tags info for category by tag name and category ID. 
 
 Examples:
   govc tags.get TAGID
   govc tags.get -n TAGNAME CATEGORYID `
 }
 
-type getTagName []tags.Tag
+type getTagInfo []tags.Tag
 
-func (t getTagName) Write(w io.Writer) error {
+func (t getTagInfo) Write(w io.Writer) error {
 	tw := tabwriter.NewWriter(w, 2, 0, 2, ' ', 0)
 
 	for _, item := range t {
-		fmt.Fprintf(tw, "Name: %s\nID: %s\nDescription: %s\nCategoryID: %s\nUsedBy: %s\n",
-			item.Name, item.ID, item.Description, item.CategoryID, item.UsedBy)
+		fmt.Fprintf(tw, "Name:\t%s\n", item.Name)
+		fmt.Fprintf(tw, "  ID:\t%s\n", item.ID)
+		fmt.Fprintf(tw, "  Description:\t%s\n", item.Description)
+		fmt.Fprintf(tw, "  CategoryID:\t%s\n", item.CategoryID)
+		fmt.Fprintf(tw, "  UsedBy: %s\n", item.UsedBy)
 	}
```

Comment:
Use separate lines for Fprintf for readable and match the style with other govc/*/.info